lib/puppet/provider/cisco_vni/nxapi.rb

```diff
@@ -1,135 +0,0 @@
-# The NXAPI provider for cisco vni.
```

Comment:
@chrisvanheuveln made some changes to NU vni.rb to distinguish mt_full and mt_lite. Not sure if its being used somewhere.